addons/website_event_track/models/event_event.py

```diff
@@ -50,6 +55,43 @@ def _compute_sponsor_count(self):
         for event in self:
             event.sponsor_count = result.get(event.id, 0)
 
+    @api.depends('event_type_id', 'website_menu')
+    def _compute_website_track(self):
+        for event in self:
+            if event.event_type_id and event.event_type_id != event._origin.event_type_id:
+                event.website_track = event.event_type_id.website_track
+            elif not event.website_menu or not event.website_track:
+                event.website_track = False
+
+    @api.depends('event_type_id', 'website_track')
+    def _compute_website_track_proposal(self):
+        for event in self:
+            if event.event_type_id and event.event_type_id != event._origin.event_type_id:
+                event.website_track_proposal = event.event_type_id.website_track_proposal
+            elif not event.website_track:
+                event.website_track_proposal = False
+
+    @api.onchange('website_track_proposal')
+    def _onchange_website_track_proposal(self):
+        """ Keep an explicit onchange for tick / untick of website_track_proposal.
+        Indeed otherwise you have a loop of dependencies between website_track and
+        website_track_proposal
+
+          * untick website_track: website_track_proposal = False (done in _compute_website_track_proposal)
+          * tick website_track: no effect
+          * untick website_track_proposal: no effect
+          * tick website_track_proposa: website_track = True
+
+        It would be complicated to write in computed fields, as they depend on
+        each other, on cache and current values, ...
+
+        It is therefore simpler to keep an onchange: when ticking website_track_proposal
+        set website_track as True in interface.
+        """
+        for event in self:
+            if event.website_track_proposal and not event.website_track:
+                event.website_track = event.website_track_proposal
```

Comment:
Maybe explicitly set it to True for clarity?

Reply:
This is not wrong.